src/NServiceBus.Core.Tests/Pipeline/Outgoing/OutgoingReplyContextTests.cs

```diff
@@ -46,5 +46,23 @@ public void ShouldShallowCloneContext()
             Assert.AreEqual("updatedValue", updatedValue);
             Assert.AreEqual("anotherValue", anotherValue2);
         }
+
+
+        [Test]
+        public void ShouldNotMergeOptionsToParentContext()
```

Comment:
shouldn't we have that test for the send, subscript and unsubscribe contexts too?